src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj

```diff
@@ -1,41 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
```

Comment:
Can you change the tab to space and remove unnecessary changes please.

Reply:
my vs code auto format like this. it should be fine

Reply:
no it's not fine, all the unnecessary changes creates a lot of merge conflict for someone who is currently working on the same file. We should keep our commit change as clean and tide as possible.